Neos.ContentRepository.Core/Classes/Projection/ContentStream/ContentStreamProjection.php

```diff
@@ -160,14 +159,13 @@ public function canHandle(EventInterface $event): bool
                 ContentStreamWasRemoved::class,
                 DimensionShineThroughWasAdded::class,
             ])
-            || $event instanceof EmbedsContentStreamAndNodeAggregateId;
+            || $event instanceof EmbedsContentStreamId;
```

Comment:
Shouldn't this be
```suggestion
            || ($event instanceof EmbedsContentStreamId && $event instanceof EmbedsNodeAggregateId)
```
?

Reply:
Yes, if you want to replace it 1:1. But I the code does not need the NodeAggreageId, so I'd rather drop this.